### DISCUSSION TITLE
How to get the default neovim cmdline?

### DISCUSSION THREAD

**Discussion by T1CkShot** [LazyVim/LazyVim#1275] [Q&A] [answered]
Hello, I've trying to change the noice.nvim cmdline to regular neovim cmdline.

Here's what I've tried,
```lua
-- plugins/noice.lua
return {
  "folke/noice.nvim",
  event = "VeryLazy",
  opts = function()
    return {
      cmdline = {
        enabled = false, -- enables the Noice cmdline UI
        view = "cmdline", -- view for rendering the cmdline. Change to `cmdline` to get a classic cmdline at the bottom
      }
    }
  end
}
```
This completely removes the cmdline from the view but the command does get inputed.

If I set `enabled = true` and `view = 'cmdline'` then the view does switch to bottom but instead of there being a permanent cmdline under statusline the statusline gets replaced while typing command.

**Comment by dpetka2001** [comment, marked as answer]
Try this
```lua
  {
    "folke/noice.nvim",
    opts = {
      cmdline = {
        enabled = false,
      },
      messages = {
        enabled = false,
      },
    },
  },
```
It states in the docs
> messages = {
    -- NOTE: If you enable messages, then the cmdline is enabled automatically.
    -- This is a current Neovim limitation.
    enabled = true, -- enables the Noice messages UI

Edit: actual answer taken from this [noice issue](https://github.com/folke/noice.nvim/issues/560).

**Reply by T1CkShot** [reply]
Thank you! This worked.
